.circleci/config.yml

```diff
@@ -49,34 +67,51 @@ jobs:
   test:
     docker:
       - image: python:3-buster
+        auth: *docker-hub-auth
     steps:
       - checkout
-      - install-python-dev-tools
+      - pipenv-sync
+      - run:
+          name: nose2
+          command: pipenv run nose2 -v
+  test-skip-lock:
+    docker:
+      - image: python:3-buster
+        auth: *docker-hub-auth
+    steps:
+      - checkout
+      - pipenv-install-skipping-lock
       - run:
           name: nose2
           command: pipenv run nose2 -v
 
   build-api-doc:
     docker:
       - image: python:3-buster
+        auth: *docker-hub-auth
     steps:
       - checkout
-      - install-python-dev-tools
+      - pipenv-sync
       - run:
           name: sphinx-build
           command: |
             pipenv run sphinx-build docs build/sphinx/html
+            tar cvjf build/sphinx/html.tar.bz2 -C build/sphinx html
       - persist_to_workspace:
           root: build/sphinx
           paths:
             - html
+      - store_artifacts:  # for reviewers to check generated doc
+          path: build/sphinx/html.tar.bz2
+          destination: html.tar.bz2
```

Comment:
plz store `html` dir. If it is archived like this, we must download it to view.

Reply:
I didn't know we can view HTML artifacts rendered in browser (right?).
OK, I will store only html dir.

Reply:
82c3c07
🎉 https://310-321225649-gh.circle-artifacts.com/0/html/index.html

Reply:
working https://310-321225649-gh.circle-artifacts.com/0/html/index.html